src/main/java/frc/robot/components/Pigeon.java

```diff
@@ -45,6 +45,10 @@ public void reset() {
         setYaw(0);
     }
 
+    public ErrorCode resetGyroWithErrorCode() {
+        return setYaw(0);
+    }
```

Comment:
we can just make reset() return ErrorCode. I don't see why would we need two separate pigeon reset methods